server/views/applications/pages/risk-of-serious-harm/risk-to-others.njk

```diff
@@ -1,11 +1,54 @@
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
 {% extends "./_risk-of-serious-harm-screen.njk" %}
+{% from "../../../partials/dateOfOasysImport.njk" import dateOfOasysImport %}
+
 {% set pageName = "risk-to-others" %}
 
 {% block questions %}
-  <h1 class="govuk-heading-l">{{ page.title }}</h1>
+  {% if page.importDate %}
+    {% set date = page.importDate %}
```

Comment:
does this need to be set explicitly or can `page.importDate` be handed into `dateOfOasysImport`?